java/openjdk21/files/JDK-8266242-undecl-ident-nsbun-arm64-workaround.patch

```diff
@@ -0,0 +1,20 @@
+--- src/java.desktop/macosx/native/libawt_lwawt/awt/CGraphicsDevice.m.orig	2024-07-09 18:07:16.000000000 +1000
++++ src/java.desktop/macosx/native/libawt_lwawt/awt/CGraphicsDevice.m	2024-10-08 18:34:39.000000000 +1100
+@@ -60,6 +60,9 @@
+ }
+ 
+ static BOOL isValidDisplayMode(CGDisplayModeRef mode) {
++    // https://trac.macports.org/ticket/71049: temporary additional guard for undef'd NSBun..ARM64
++    // Should be reported/fixed upstream at openjdk.org.
```

Comment:
It has already been reported, right? https://bugs.openjdk.org/browse/JDK-8266242

Reply:
Per comment on openjdk17 patch.